packages/flutter/lib/src/material/drawer_header.dart

```diff
@@ -51,20 +91,93 @@ class DrawerHeader extends StatelessWidget {
           )
         )
       ),
-      child: new Stack(
-        children: <Widget>[
-          background ?? new Container(),
-          new Positioned(
-            top: statusBarHeight + 16.0,
-            left: 16.0,
-            right: 16.0,
-            bottom: 8.0,
-            child: new DefaultTextStyle(
-              style: Theme.of(context).textTheme.body2,
-              child: content
+      child: new Container(
+        padding: const EdgeInsets.only(left: 16.0, right: 16.0, bottom: 8.0),
+        decoration: config.decoration ?? new BoxDecoration(),
+        child: new Column(
+          children: <Widget>[
+            new Flexible(
+              child: new Stack(
+                children: <Widget>[
+                  new Positioned(
+                    top: statusBarHeight + 16.0,
+                    right: 0.0,
+                    child: new Row(
+                      children: config.otherAccountsPictures.map(
+                        (p) => new Container(
+                          margin: const EdgeInsets.only(left: 16.0),
+                          constraints: new BoxConstraints.tight(
+                              new Size(42.0, 42.0)),
```

Comment:
Use const.

Only indent two spaces and move the closing bracket to the following line:
```
constraints: new BoxConstraints.tight(
  const Size(42.0, 42.0)
)
```
When you've reached the leaves of the build function's widget tree, it's OK to just put everything on one line if it's small and simple:
```
constraints: new BoxConstraints.tight(const Size(42.0, 42.0))
```